datapacks/fleet/entities/battleship.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Battleship",
+    "max_health": 200,
```

Comment:
Warning: This health stuff will change in future, so expect this to break. It's fine for now but it'll be moved